lighthouse-core/test/results/sample_v2.json

```diff
@@ -5587,31 +5587,31 @@
       },
       {
         "startTime": 0,
-        "name": "lh:audit:redirects",
+        "name": "lh:computed:PageDependencyGraph",
```

Comment:
all that's happening here is PageDependencyGraph is moving before the others now that we request it as part of `critical-request-chains`